.travis.yml

```diff
@@ -11,6 +11,9 @@ matrix:
     - python: 3.7
```

Comment:
>- python: 3.7 [](start = 4, length = 13)

remove ?

Reply:
No, we don't want to remove any of the previous versions. We want the test runs to cover all major Python versions, as seen [here](https://travis-ci.org/AzureAD/microsoft-authentication-library-for-python/builds/598305717?utm_source=github_status&utm_medium=notification).